### DISCUSSION TITLE
建议增加和谷歌翻译一样的输入框，可以单独复制单词、句子、段落进行翻译

### DISCUSSION THREAD

**Discussion by gedageda** [immersive-translate/immersive-translate#403] [Ideas]
### 新功能 | New Feature

1.建议增加和谷歌翻译一样的输入框，可以单独复制单词、句子、段落进行翻译。这样基本就完全取得原谷歌翻译了

### 为什么您想要添加这些功能 | Why you want to add these features

1.使用的时候更方便！
2.
3.

**Comment by versun** [comment]
正在开发中,关注这个哈:https://github.com/immersive-translate/immersive-translate/discussions/377

**Reply by theowenyoung** [reply]
这似乎不是同一个功能，这个是允许用户自己输入段落～

**Reply by versun** [reply]
哈哈, 没看清楚, 不过感觉目的是一样的, 直接划词翻译更方便吧

**Reply by gedageda** [reply]
不是一个功能，难道没用过谷歌翻译么？

**Reply by chieh0958** [reply]
有时候是自己OCR来的文章文本或是其他非网页来源，会需要一个可以自行输入的文本框
